tests/pthread/test_pthread_locale.c

```diff
@@ -52,10 +52,7 @@ int main (int argc, char *argv[])
     pthread_join(thread, (void**)&child_loc);
   }
 
-#ifdef REPORT_RESULT
-  int result = (main_loc == child_loc);
-  REPORT_RESULT(result);
-#endif
 
-  pthread_exit(NULL);
+  assert(main_loc == child_loc);
```

Comment:
Why is this necessary? Isn't the `return` below gonna signal whether the test succeeds or not anyway?

Reply:
You are right I will just stick to one or the other.